src/HID-APIs/AbsoluteMouseAPI.hpp

```diff
@@ -78,8 +78,8 @@ void AbsoluteMouseAPI::moveTo(int x, int y, signed char wheel){
 	yAxis = y;
 	HID_MouseAbsoluteReport_Data_t report;
 	report.buttons = _buttons;
-	report.xAxis = x;
-	report.yAxis = y;
+	report.xAxis = ((long)x + 32768) / 2;
```

Comment:
instead of long, please use uint32_t. I am also not use, if the number after should be castes as well. It would make sense to use a 0x?? like representation to understand it better. Please also add a comment that we are converting signed range (xx -yy) to unsigned range 0-0xFFFF

Reply:
> instead of long, please use uint32_t

Okay, but int32_t, when calculating, we still need a sign.

> 0x

Just for uniformity to avoid "stairs".

Also comment added.